README.md

```diff
@@ -43,13 +53,20 @@ You can start editing the page by modifying `pages/index.js` (we talk about `ind
 The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
 
 
-### Repo's structure
+### How to run the built site in user mode
 
-Into this repo you will find some folders:
+For displaying the built site in your browser an host static HTTP server is needed. 
+One possibility is to use the [`python` one](https://docs.python.org/3/library/http.server.html), but this option can be too poor in such cases, in which where it may happen that some pages are not rendered properly. 
```

Comment:
```suggestion
One possibility is to use the [`python` one](https://docs.python.org/3/library/http.server.html), but this option can be too poor in such cases, in which it may happen that some pages are not rendered properly. 
```